vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -577,6 +578,51 @@ func resourceVSphereComputeCluster() *schema.Resource {
 					},
 				},
 			},
+			"vsan_stretched_cluster_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Whether the vSAN stretched cluster is configured for the cluster.",
+			},
+			"vsan_stretched_cluster_conf": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Computed:    true,
+				Description: "The configuration for stretched cluster.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"preferred_fault_domain_host_ids": {
+							Type:        schema.TypeSet,
+							Optional:    true,
+							Description: "The managed object IDs of the hosts to put in the first fault domain.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+						"secondary_fault_domain_host_ids": {
+							Type:        schema.TypeSet,
+							Optional:    true,
+							Description: "The managed object IDs of the hosts to put in the second fault domain.",
+							Elem:        &schema.Schema{Type: schema.TypeString},
+						},
+						"witness_node": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The managed object IDs of the host selected as witness node when enable stretched cluster.",
+						},
+						"preferred_fault_domain_name": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The name of prepferred fault domain.",
+							Default:     "Preferred",
```

Comment:
I assume these are sane/expected default values vSphere typically uses/assumes? Or does it really not matter.

Reply:
> I assume these are sane/expected default values vSphere typically uses/assumes? Or does it really not matter.

Yes, these two names are default values, but the name of fault domains are mandatory.